src/plugins/ucx/ucx_backend.cpp

```diff
@@ -463,6 +466,12 @@ nixlUcxEngine::nixlUcxEngine (const nixlBackendInitParams* init_params)
             this->initErr = true;
             return;
         }
+
+        // This will ensure that the resulting delay is at least 1ms and fits into int in order for
+        // it to be compatible with poll()
+        pthrDelay = std::chrono::ceil<std::chrono::milliseconds>(
```

Comment:
probably worth changing description of this var in `nixlAgentConfig`, because it has different meaning now

Reply:
ack